src/link/patch.c

```diff
@@ -111,7 +111,7 @@ static uint32_t getRPNByte(uint8_t const **expression, int32_t *size,
 static struct Symbol const *getSymbol(struct Symbol const * const *symbolList,
 				      uint32_t index)
 {
-	assert(index != -1); /* PC needs to be handled specially, not here */
+	assert(index != (uint32_t)-1); /* PC needs to be handled specially, not here */
```

Comment:
`UINT32_MAX`, perhaps?

Reply:
The docs mention "-1" specifically, so I prefer this, though not by much.

Reply:
It's used a few times in output.c as well; how about a `PC_BANK_MARKER` constant?

Reply:
It's used in all of those places as a "no value" marker, so it'd need to be more generic than that.

Reply:
I see, they aren't all for "BANK(symbol) where -1 means PC"; never mind then.